README.md

```diff
@@ -9,8 +9,31 @@ scale-up, operation and troubleshooting of innovative, advanced energy systems.
 ![Tests](https://github.com/IDAES/idaes-pse/workflows/Tests/badge.svg?branch=main)
 ![Integration](https://github.com/IDAES/idaes-pse/workflows/Integration/badge.svg?branch=main)
 [![codecov](https://codecov.io/gh/IDAES/idaes-pse/branch/main/graph/badge.svg?token=1lNQNbSB29)](https://codecov.io/gh/IDAES/idaes-pse)
+[![Documentation Status](https://readthedocs.org/projects/idaes-pse/badge/?version=latest)](https://idaes-pse.readthedocs.io/en/latest/?badge=latest)
 <!-- END Status badges -->
 
+## Quickstart

```

Comment:
I think the quickstart should point to full installation instructions, since particularly the pip version won't get the users to a point where most of the models can be run. Also, usually a quickstart ends with running something to see that it worked. This is more of a "quick-install".

Reply:
Ok, I tried to re-work things to make it clearer.